homeassistant/components/shelly/climate.py

```diff
@@ -98,19 +98,14 @@ async def async_restore_climate_entities(
         ent_reg, config_entry.entry_id
     )
 
-    restore_entries = []
     for entry in entries:
 
         if entry.domain != CLIMATE_DOMAIN:
             continue
 
         _LOGGER.debug("Setup sleeping climate device %s", wrapper.name)
         _LOGGER.debug("Found entry %s [%s]", entry.original_name, entry.domain)
-
-        restore_entries.append(BlockSleepingClimate(wrapper, None, None, entry))
-
-    if restore_entries:
-        async_add_entities(restore_entries)
+        async_add_entities([BlockSleepingClimate(wrapper, None, None, entry)])
```

Comment:
Let's add a break after this so we don't continue the iteration in vain and make it more clear what's going on.
```suggestion
        async_add_entities([BlockSleepingClimate(wrapper, None, None, entry)])
        break
```

Reply:
Done